### PR TITLE
Fix Docker images

### PR DESCRIPTION
**Pull request by jangrewe**
This decreases the image size from nearly 900 MB to around 10MB and fixes the magneticow image (go-bindata error).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,11 +21,13 @@ image: image-magneticod image-magneticow
 # Download dependencies
 ensure:
 	dep ensure -v
+	go get -u github.com/jteeuwen/go-bindata/...
```
Comment (boramalper) [inline Makefile]: Nice! But jteeuwen/go-bindata is unmaintained, so I use [github.com/kevinburke/go-bindata/](https://github.com/kevinburke/go-bindata/) instead.

```suggestion
	go get -u github.com/kevinburke/go-bindata/...
```

File: Makefile [hunk 2]
```diff
@@ -21,11 +21,13 @@ image: image-magneticod image-magneticow
...
 
 test-magneticod:
 	go test github.com/boramalper/magnetico/cmd/magneticod/...
 
 test-magneticow:
```
Comment (boramalper) [inline Makefile]: `test-*` cases are removed in v0.7.0, we use `go test github.com/boramalper/magnetico/...` instead. If you can update your PR for that version, it'd be great. =)

Comment (jangrewe) [reply Makefile]: Thanks, i've updated my PR!

**Comment by boramalper** [comment]
Congrats on becoming a contributor! =)

**Pull request merged**